svc/shows/endpoint.go

```diff
@@ -13,17 +13,37 @@ import (
 type (
 	// Endpoints collection of profile service
 	Endpoints struct {
+		AddShow            endpoint.Endpoint
+		DeleteShowByID     endpoint.Endpoint
 		GetShows           endpoint.Endpoint
 		GetShowChallenges  endpoint.Endpoint
 		GetShowByID        endpoint.Endpoint
 		GetShowsByCategory endpoint.Endpoint
+		UpdateShow         endpoint.Endpoint
+
+		AddEpisode            endpoint.Endpoint
+		DeleteEpisodeByID     endpoint.Endpoint
+		DeleteEpisodeByShowID endpoint.Endpoint
```

Comment:
is this method required? it would delete all episodes of concrete show - therefore it's more useful to make episodes cascade to show and delete show would lead to deleting all it's episodes. @dmitrymomot what do you think?

Reply:
yes, it makes sense, given that these two entities are in one service